EliteDangerous/JournalEvents/JournalScan.cs

```diff
@@ -487,13 +487,25 @@ public string DisplayString(int indent = 0, bool includefront = true)
             return scanText.ToNullSafeString().Replace("\n", "\n" + inds);
         }
 
+        public string GetHabZoneStringLs()
+        {
+            if (IsStar && HabitableZoneInner.HasValue && HabitableZoneOuter.HasValue)
+            {
+                return $"{HabitableZoneInner.Value.ToString("N0")}-{HabitableZoneOuter.Value.ToString("N0")}ls";
+                    }
```

Comment:
I think the brace is getting attacked by space invaders! 👾  (trim the extra indentation, please)